latest/ug/nodes/hybrid-nodes-networking.adoc

```diff
@@ -392,7 +392,15 @@ The following access for your EKS cluster security group is required for ongoing
 |Kubernetes API server to webhook (if running webhooks on hybrid nodes)
 |===
 
-To create a security group with the inbound access rules, run the following commands. This security group must be passed when you create your Amazon EKS cluster. By default, the command below creates a security group that allows all outbound access. You can restrict outbound access to include only the rules above. If you're considering limiting the outbound rules, we recommend that you thoroughly test all of your applications and pod connectivity before you apply your changed rules to a production cluster.
+[IMPORTANT]
+* **Security group rule limits**: Amazon EC2 security groups have a maximum of 60 ingress rules by default. The security group ingress rules may not apply if your cluster security group approaches this limit. In this case, it may be required to manually add in the missing ingress rules.
+* **CIDR cleanup responsibility**: When you update your cluster to remove remote node or pod CIDRs, Amazon EKS does not automatically remove the corresponding security group rules. You are responsible for manually removing unused CIDR rules to prevent reaching the 60-rule limit.
```

Comment:
This isn't just related to the 60-rule limit. Suggest rephrasing to

* **CIDR cleanup responsibility**: If you remove remote node or pod networks from EKS clusters, EKS does not automatically remove the corresponding security group rules. You are responsible for manually removing unused remote node or pod networks from your security group rules.